tests/test_polygon.py

```diff
@@ -104,6 +104,10 @@ def test_linearring_coords_numpy():
     expected = np.asarray([(0.0, 0.0), (0.0, 1.0), (1.0, 1.0), (0.0, 0.0)])
     assert_array_equal(ra, expected)
 
+
+def test_numpy_empty_linearring_coords():
+    np = pytest.importorskip("numpy")
```

Comment:
Yes, this is fine, too. It seems like standardizing on one form of the other of skipping without numpy would have benefits, but I don't want to belabor this PR. I'm okay with a mix of `importorskip` and `skipif(not numpy)`. @jorisvandenbossche do you have a strong preference?

Reply:
@sgillies agreed that it's certainly not ideal, basically I just tried to follow the usage elsewhere in the file. Perhaps in a separate PR I could work on transitioning all of the tests to pytest style?

Reply:
I have a preference to use pytest whenever we write new tests. 

Doing a large conversion to pytest style might also be welcome, but might maybe be most useful if it directly refactors the tests (like splitting up more, better coverage, .. like eg what I did in https://github.com/Toblerity/Shapely/pull/963). There are also a lot of `self.assertEqual(..)`s to clean up ..

Reply:
BTW, specifically for numpy, it's also going to be a requied dependency for 2.0, so then we won't need those skips anymore.